sci-log-db/Dockerfile

```diff
@@ -1,9 +1,13 @@
 # Check out https://hub.docker.com/_/node to select a new base image
-FROM node:14-slim
+FROM node:18-slim
 
 # Install extra packages
 RUN apt-get update && apt-get install -y zip \
- texlive-latex-extra
+ texlive-latex-extra && \
```

Comment:
I guess latex is not needed anymore?

Reply:
correct, thanks for noticing that! I will remove it in the next commit